dds/DCPS/DataReaderImpl.cpp

```diff
@@ -224,13 +222,12 @@ DataReaderImpl::cleanup()
 }
 
 void DataReaderImpl::init(
-    TopicDescriptionImpl* a_topic_desc,
+    TopicDescriptionImpl*       a_topic_desc,
```

Comment:
The coding guidelines "...Do not add extra spaces to make syntax elements (that span lines/statements) line up. This only causes unnecessary changes in adjacent lines as the code evolves."

If we're going to make changes anyway, I'd rather change it to match the guidelines.  The other option is to leave the code as-is.